index.js

```diff
@@ -46,7 +46,7 @@ function filterResponseForImportantEvents(allEventsFromFetch) {
 }
 
 function fetchPageOfDataAndFilter(url) {
-    return new Promise((resolve) => {
+    return new Promise((resolve, reject) => {
```

Comment:
Eventually I would like to see this function switched to be async, and using throw, rather than the (resolve, reject) style. 

But I view that as out of scope for this PR, so no action required at this time.